lib/Backend/Inline.cpp

```diff
@@ -1010,6 +1010,13 @@ Inline::InlinePolymorphicFunctionUsingFixedMethods(IR::Instr *callInstr, const F
     return instrNext;
 }
 
+IR::RegOpnd * Inline::GetCallbackFunctionOpnd(IR::Instr * callInstr)
+{
+    IR::Instr * callbackSymDefInstr = callInstr->GetSrc1()->GetStackSym()->GetInstrDef();
```

Comment:
>callbackSymDefInstr [](start = 16, length = 19)

Can you rename this to, say, callApplyLdInstr? Its not really the def instr for the callback, is it?

Reply:
Renamed. It's not the def instr for the callback, not sure why I named it as such.